react-example/src/Hooks/useLocalStorage/useLocalStorage.js

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect} from "react";
```

Comment:
In this case, being a single file, I think the use of the folder `useLocalStorage` doesn't make much sense, you can just place the file inside the `Hooks` folder